Mathlib/Combinatorics/SimpleGraph/Subgraph.lean

```diff
@@ -669,6 +672,9 @@ theorem comap_monotone {G' : SimpleGraph W} (f : G →g G') : Monotone (Subgraph
     intro
     apply h.2
 
+@[simp] lemma comap_equiv_top {H : SimpleGraph W} (e : G ≃g H) : Subgraph.comap e.toHom ⊤ = ⊤ := by
+  ext <;> simp [Relation.Map, e.apply_eq_iff_eq_symm_apply, ← e.map_rel_iff]
```

Comment:
(there's a slightly silly proof of this using `map_le_iff_le_comap` with H' := top and uniqueness of top...)